types/regen.sh

```diff
@@ -58,6 +59,7 @@ for up in "${!filename_map[@]}"; do
   cat $tmpdir/$UPSTREAM_SUBDIR/$up |
     # Adjust proto package.
     # TODO(dsymonds): Upstream the go_package option instead.
+    grep -v '^\s*option\s\+go_package' |
```

Comment:
This hack can similarly be removed once we consistently have `go_package` set in the upstream `.protof` files